core/frontend/src/IModelConnection.ts

```diff
@@ -612,6 +617,27 @@ export abstract class IModelConnection extends IModel {
     }
     return ("number" === typeof this._projectCenterAltitude) ? this._projectCenterAltitude : undefined;
   }
+
+  /**
+   * Gets the context that allows accessing the metadata (ecschema-metadata package) of this iModel.
+   * The context is created lazily when this property is accessed for the first time, with an `ECSchemaRpcLocater` registered.
+   * This means to correctly access schema context, client-side applications must register ECSchemaRpcInterface following instructions for [RPC configuration](https://www.itwinjs.org/learning/rpcinterface/#client-side-configuration).
+   * Server-side applications would also [configure RPC](https://www.itwinjs.org/learning/rpcinterface/#server-side-configuration) as needed.
+   *
+   * @note `BlankConnection` `schemaContext` will throw an error when trying to access schema metadata.
```

Comment:
```suggestion
   * @note The getter throws an error when called on `BlankConnection`.
```

Reply:
The getter doesn't throw an error, my note might not have been clear enough. I edited it to mention blankConnection does return a valid schemacontext, but has an invalid locater registered and will throw errors when calling its methods